Makefile

```diff
@@ -120,30 +129,33 @@ unit:
 
 .PHONY: image
 image: ## Build docker image
+ifeq ($(NO_DOCKER), 1)
+	./hack/imagebuilder.sh
+endif
 	@echo -e "\033[32mBuilding image $(IMAGE):$(VERSION) and tagging also as $(IMAGE):$(MUTABLE_TAG)...\033[0m"
 	$(IMAGE_BUILD_CMD) -t "$(IMAGE):$(VERSION)" -t "$(IMAGE):$(MUTABLE_TAG)" ./
 
 .PHONY: push
 push: ## Push image to docker registry
 	@echo -e "\033[32mPushing images...\033[0m"
-	docker push "$(IMAGE):$(VERSION)"
-	docker push "$(IMAGE):$(MUTABLE_TAG)"
+	$(ENGINE) push "$(IMAGE):$(VERSION)"
+	$(ENGINE) push "$(IMAGE):$(MUTABLE_TAG)"
 
 .PHONY: lint
 lint: ## Go lint your code
-	hack/go-lint.sh -min_confidence 0.3 $(go list -f '{{ .ImportPath }}' ./...)
+	 $(DOCKER_CMD) hack/go-lint.sh -min_confidence 0.3 $(go list -f '{{ .ImportPath }}' ./...)
 
 .PHONY: fmt
 fmt: ## Go fmt your code
-	hack/go-fmt.sh .
+	$(DOCKER_CMD) hack/go-fmt.sh .
 
 .PHONY: goimports
 goimports: ## Go fmt your code
```

Comment:
Suggest to fix this docstring.
Something like `Updates and shows diff for import lines`